### PR TITLE
Add README and english description in metadata.txt

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: PreCourlis/metadata.txt [hunk 1]
```diff
@@ -10,12 +10,12 @@
 name=PreCourlis
 qgisMinimumVersion=3.0
 qgisMaximumVersion=3.99
-description=Creation de profils pour Courlis
+description=Create layered cross sections of rivers for Courlis
```
Comment (msecher) [inline PreCourlis/metadata.txt:13]:
```suggestion
description=Create geometry - sediment layered 1D cross-sections - of rivers for COURLIS which is the sediment transport module for MASCARET, the 1D free surface flow code of the Open TELEMAC-MASCARET platform (http://www.opentelemac.org/)
```

File: PreCourlis/metadata.txt [hunk 2]
```diff
@@ -10,12 +10,12 @@
...
 version=2.0
 author=EDF Hydro, DeltaCAD, Camptocamp
 email=matthieu.secher@edf.fr
 
-about=Creation de profils pour Courlis
+about=This plugin allows you to model the bed of rivers and export data for the Courlis module of Telemac-Mascaret
```
Comment (msecher) [inline PreCourlis/metadata.txt:18]:
```suggestion
about=This plugin allows you to 
- import georeferenced MASCARET geometry, 
- extract geometric information from a DEM (raster) on polylines representing the cross-sections of the MASCARET-COURLIS model in respect with a user defined discretization, 
 -  interpolate cross sections along an hydraulic axis in respect with user defined longitudinal (number of cross-sections) and lateral (number of points on each profile) discretization, 
 - visualize the cross-sections in a graphical view,
 - adding sediment layers and bed rock to the cross-sections from river bed geometry (identical or with a Z shift) or from another DEM (raster),
 - modified manually or with some functionality more automatic the river bed geometry, sediment layers geometry or bedrock geometry,
 - and export data for the COURLIS module

PRECOURLIS is using :
- A fork of TatooineMesher (courtesy of CNR Engineering, https://github.com/CNR-Engineering/TatooineMesher)
- Modified version of Python classes of the TELEMAC-MASCARET system

```

**Review by msecher — COMMENTED** [review]
I suggest you some modification in the "description" and "about" sections.

I don't know if there is a limit of length and if we are allowed to have multiple lines, but I think it could be great to describe the main functionalities of PRECOURLIS and show them in an easy-to-read format

**Comment by arnaud-morvan** [comment]
@msecher : yes it is possible to set mutiple lines texts in using indentation.
I fixed your about text accordingly.

**Pull request merged**